addon/mixins/sl-tooltip-enabled.js

```diff
@@ -98,11 +93,10 @@ export default Ember.Mixin.create({
     /**
      * Enable tooltip
      *
-     * @private
```

Comment:
Seems this would still be applicable, denoting that it is a method used internal to the object to assist with other logic, but not intended to be called/executed directly.